core/assets/bundles/bundle.properties

```diff
@@ -290,6 +290,7 @@ cancelbuilding = [accent][[{0}][] to clear plan
 selectschematic = [accent][[{0}][] to select+copy
 pausebuilding = [accent][[{0}][] to pause building
 resumebuilding = [scarlet][[{0}][] to resume building
+showui = [accent][[{0}][] to show UI again
```

Comment:
How about `Press [accent][[{0}][] to show UI again`?

Reply:
should also mention "UI hidden." as the first line